Commands/ContentTypes/GetContentType.cs

```diff
@@ -64,17 +64,35 @@ protected override void ExecuteCmdlet()
                     List list = List.GetList(SelectedWeb);
 
                     ClientContext.ExecuteQueryRetry();
+
                     if (!string.IsNullOrEmpty(Identity.Id))
                     {
-                        var cts = ClientContext.LoadQuery(list.ContentTypes.Include(ct => ct.Id, ct => ct.Name, ct => ct.StringId, ct => ct.Group).Where(ct => ct.StringId == Identity.Id));
-                        ClientContext.ExecuteQueryRetry();
-                        WriteObject(cts, true);
+                        if(list.ContentTypeExistsById(Identity.Id))
+                        {
+                            var cts = list.GetContentTypeById(Identity.Id);
+                            ClientContext.Load(cts);
+                            ClientContext.ExecuteQueryRetry();
+                            WriteObject(cts, false);
+                        }
+                        else
+                        {
+                            WriteVerbose(string.Format("Content Type Id: {0} does not exist in the list: {1}", Identity.Id, List.Title));
```

Comment:
This would change the behavior of the cmdlet: where in previous versions it would throw an error, writing a verbose message would 'swallow' the exception (people have specify the -Verbose parameter to see the message). People could use this cmdlet if a content type exists, and now they will not receive an error message (which they might catch in their scripts otherwise). Possible change for this line can be:

```csharp
 WriteError(new ErrorRecord(new ArgumentException("Content Type Id: {0} does not exist in the list: {1}"), "CONTENTTYPEDOESNOTEXIST", ErrorCategory.InvalidArgument, this));
```